src/shared/vscode/user_prompts.ts

```diff
@@ -48,22 +48,22 @@ export function showFlutter2019Q2SurveyNotificationIfAppropriate(context: Contex
 		logger.warn("Unable to read Flutter settings for preparing survey link");
 	}
 
-	const prompt = clientID ? flutterSurvey2019Q2PromptWithAnalytics : flutterSurvey2019Q2PromptWithoutAnalytics;
-	const surveyUrl = "https://google.qualtrics.com/jfe/form/SV_3W3aVD2y9CoAe6V?Source=VSCode"
+	const prompt = clientID ? flutterSurveyPromptWithAnalytics : flutterSurveyPromptWithoutAnalytics;
+	const surveyUrl = "https://google.qualtrics.com/jfe/form/SV_3kiGXYfYOfXUjB3?Source=VSCode"
```

Comment:
@jayoung-lee also double-check the URL. Your example has "IDE", but I stuck with "VSCode".

Reply:
The link is correct!